tools/generators/move-packages/index.spec.ts

```diff
@@ -49,6 +49,9 @@ describe('move-packages generator', () => {
     };
     tree.write(`nx.json`, serializeJson(nxJsonConfig));
 
+    const codeOwners = `packages/test @dummyOwner`;
```

Comment:
consider leveraging this https://github.com/microsoft/fluentui/blob/master/tools/utils-testing.ts#L5